com.unity.render-pipelines.high-definition/Documentation~/hair-shader.md

```diff
@@ -7,13 +7,13 @@ Under the hood, the Hair shader is a pre-configured Shader Graph. To learn more
 
 ## Importing the Hair Sample
 
-HDRP comes with a Hair Material sample to further help you get started. To find this Sample:
-
-1. Go to **Windows > Package Manager**, and select **High Definition RP** from the package list.
-2. In the main window that shows the package's details, find the **Samples** section.
-3. To import a Sample into your Project, click the **Import into Project** button. This creates a **Samples** folder in your Project and imports the Sample you selected into it. This is also where Unity imports any future Samples into.
-4. In the Asset window, go to **Samples > High Definition RP > 11.0** and open the Hair scene. Here you will see the hair sample material set up in-context with a scene, and available for you to use.
+HDRP includes the Hair sample scene to help you get started. To find this Sample:
 
+1. Go to **Window** > **Package Manager**, and select **High Definition RP** from the package list.
+2. In the main window that shows the package's details, open the **Samples** drop down.
+3. Find **Material Samples** and select the **Import** button. This creates a **Samples** folder in your Project and imports the Sample you selected into it. This is also where Unity imports any future Samples into.
+4. In the Asset window, go to **Samples** > **High Definition RP** and select the folder that corresponds with your HDRP version. 
+5. Open **Material Samples** > **Scenes** and select the the **Hair** scene. Here you will see the hair sample material set up in-context with a scene, and available for you to use.
```

Comment:
```suggestion
5. Open **Material Samples** > **Scenes** and select the **Hair** scene. Here you will see the hair sample material set up in-context with a scene, and available for you to use.
```

Reply:
I realise you didn't change this vullet but I couldn't resist removing the extra 'the'!